spec/mxmnd_spec.rb

```diff
@@ -5,3 +5,49 @@
     expect { Mxmnd::city(nil) }.to raise_error Mxmnd::BadRequest
   end
 end
+
+describe 'Mxmnd.city response' do
```

Comment:
Ruby uses 2 spaces for indentation, you seem to be using more. 
Please install Rubocop locally and run your code through it to see some of the styling issues